uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
```

Comment:
The page objects should be separated for each page of leave and absence as each of them would have `waitForReady` Function.

-page-objects/
--ssp-la/
---leave-modal.js
---my-leave-report.js (Current file content should go here)
---my-leave-calendar.js (please move and rename `ssp-la-my-leave.js`here)